account-kit/core/src/types.ts

```diff
@@ -28,36 +32,38 @@ export type SupportedAccountTypes =
   | "ModularAccountV2";
 
 export type SupportedAccounts =
-  | LightAccount<AlchemyWebSigner, LightAccountVersion<"LightAccount">>
-  | MultiOwnerModularAccount<AlchemyWebSigner>
+  | LightAccount<AlchemySigner, LightAccountVersion<"LightAccount">>
+  | MultiOwnerModularAccount<AlchemySigner>
   | MultiOwnerLightAccount<
-      AlchemyWebSigner,
+      AlchemySigner,
       LightAccountVersion<"MultiOwnerLightAccount">
     >
-  | ModularAccountV2<AlchemyWebSigner>;
+  | ModularAccountV2<AlchemySigner>;
 
 export type SupportedAccount<T extends SupportedAccountTypes> =
   T extends "LightAccount"
-    ? LightAccount<AlchemyWebSigner>
+    ? LightAccount<AlchemySigner>
     : T extends "MultiOwnerModularAccount"
-    ? MultiOwnerModularAccount<AlchemyWebSigner>
+    ? MultiOwnerModularAccount<AlchemySigner>
     : T extends "MultiOwnerLightAccount"
-    ? MultiOwnerLightAccount<AlchemyWebSigner>
+    ? MultiOwnerLightAccount<AlchemySigner>
     : T extends "ModularAccountV2"
-    ? ModularAccountV2<AlchemyWebSigner>
+    ? ModularAccountV2<AlchemySigner>
     : never;
 
 export type AlchemyAccountsConfig = {
   store: Store;
   _internal: {
+    // if not provided, the default signer will be used
+    createSigner?: (config: ClientStoreConfig) => AlchemySigner;
```

Comment:
Maybe this internally should always be defined so that it is set when a config is initialized and we can do the `?? createWebSigner` call in `store.ts` call

```suggestion
    createSigner: (config: ClientStoreConfig) => AlchemySigner;
```